src/N98/Magento/Command/Installer/SubCommand/InstallMagento.php

```diff
@@ -144,13 +146,18 @@ public function execute()
             return $url;
         };
 
+
+        $folderName = basename($this->config->getString('installationFolder'));
+        $defaultBaseUrl = sprintf($this->commandConfig['installation']['base-url'], $folderName);
```

Comment:
This use of %s inside the value is unstable. This is not a stable implementation, more a proof of concept.